apps/dapp/src/types/typechain/factories/GenericZap__factory.ts

```diff
@@ -1142,7 +1142,7 @@ const _abi = [
 ];
 
 const _bytecode =
```

Comment:
interesting that the typechain files were updated?

Reply:
It may only be the embedded metadata at the end of the bytecode that changed... Perhaps different yarn/hardhat config?

For now to unblock this, I'd remove from the pr as we know what's there works. I can check what might have changed